style.css

```diff
@@ -0,0 +1,92 @@
+* {
+  margin: 0;
+}
+
+.gen-container {
+  width: 375px;
+  height: 718px;
+  background-color: #ebebff;
+}
+
+.content {
+  width: 375px;
+  height: 718px;
+  background-color: #fff;
+  border-radius: 1px 1px 1px 40px;
+  background-image: url('./header-section/header-mobile-bg.svg');
+  background-repeat: no-repeat;
+}
```

Comment:
- [OPTIONAL] I think is a good idea to make your mobile version responsive until the desktop screen size.